core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1352,6 +1352,17 @@ public static boolean isValidZooPropertyKey(String key) {
         || key.startsWith(REPLICATION_PREFIX.getKey());
   }
 
+  /**
+   * Checks if the given sampler option is valid
+   *
+   * @param option
+   *          property option
+   * @return true if option is a valid sampler option
+   */
+  public static boolean isValidSamplerOption(String option) {
+    return option.endsWith("hasher") || option.endsWith("modules");
+  }
+
```

Comment:
Sampler options are arbitrary, and implementation-specific. The available options depend on the user's provided sampler implementation. They do not have to only be one of these options. These are merely example properties placed in the docs that an implementation might have.

Reply:
That is what I was afraid of. Guess there isn't an easy way to guarantee validity.